log/logClient.go

```diff
@@ -271,15 +272,24 @@ func watchAlertsHelper(res *pb.Alert, o Options) error {
 
 		str = fmt.Sprintf("== Alert / %s ==\n", updatedTime)
 
-		str = str + fmt.Sprintf("Cluster Name: %s\n", res.ClusterName)
-		str = str + fmt.Sprintf("Host Name: %s\n", res.HostName)
+		// Check if docker daemon is running
+		if _, err := os.Stat("/var/run/docker.pid"); err == nil {
+			dockerenabled = true // docker is running
+		}
+
+		if !dockerenabled {
+			str = str + fmt.Sprintf("Cluster Name: %s\n", res.ClusterName)
+			str = str + fmt.Sprintf("Host Name: %s\n", res.HostName)
+		}
 
 		if res.NamespaceName != "" {
```

Comment:
NamespaceName should be empty for our docker container logs right?

Reply:
No, we have set "container_namespace" for un-orchestrated containers